.github/workflows/push_centos_build_image.yml

```
@@ -0,0 +1,42 @@
+name: Build and push centos7 base image
+
+# workflow based on https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages
+on:
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}-centos7-build-image
```

why not simply:

```suggestion
  IMAGE_NAME: centos7-build-image
```

?

Also this variable could go to the `job` level instead of whole workflow.

---

Moved env to job scope in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2174/commits/6ca6e54a57992a23f627815d73c1eb3dd2f39c47